modules/appeals_api/spec/requests/v2/higher_level_reviews_controller_spec.rb

```diff
@@ -35,7 +35,7 @@ def new_base_path(path)
         uuid_2 = create(:higher_level_review_v2, veteran_icn: '1013062086V794840').id
         create(:higher_level_review_v2, veteran_icn: 'something_else')
 
-        get(path, headers: @minimum_required_headers)
+        get(path, headers: @headers_extra)
```

Comment:
It looks like the extra headers include a few more fields than just the ICN (like non-veteran claimant fields) - is it worthwhile to keep using the `@minimum_required_headers` fixture with just an ICN added? (or even add the ICN to the minimum required headers fixture?)